src/tests/execute/approve_ask_tests.rs

```diff
@@ -279,8 +280,8 @@ mod approve_ask_tests {
               \"supply_fixed\": false
             }";
 
-        let test_marker: Marker = from_binary(&Binary::from(marker_json)).unwrap();
-        deps.querier.with_markers(vec![test_marker]);
+        let _test_marker: MarkerAccount = from_binary(&Binary::from(marker_json)).unwrap();
+        // deps.querier.with_markers(vec![test_marker]); // TODO: find alternative function
```

Comment:
this PR is handling, please check https://github.com/FigureTechnologies/digital-currency-consortium/pull/159/files